tests/app/authentication/test_authentication.py

```diff
@@ -381,7 +381,7 @@ def test_proxy_key_on_admin_auth_endpoint(notify_api, check_proxy_header, header
 
 class TestRequiresUserInService:
 
-    def test_accepts_jwt_for_user_in_service(self, client, db_session):
+    def test_accepts_jwt_for_user_in_service(self, client, notify_db_session):
```

Comment:
The "notify_db_session" fixture has function scope and takes the "db_session" fixture as a parameter.  The latter has session scope.

Reply:
Should you create a ticket to add it back in?

Reply:
#1106